src/main/java/tric/tricproject/Controller/AdminController.java

```diff
@@ -145,6 +145,17 @@ public ResponseEntity<Status> setActive() {
         }
     }
 
+    @PostMapping("/deactivate")
+    public ResponseEntity<Status> setInactive() {
+        try {
+            Status status = statusService.setAppStatus(false);
```

Comment:
can just set the status, you don't need the 'Status status =' part

Reply:
Shouldn't I send back the response with return new ResponseEntity<>(status, HttpStatus.CREATED);?